src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -125,10 +135,7 @@ SGVector<T> from_gpu(const SGVector<T>& vector)
 		return SGVector<T>(data, vector.vlen);
 	}
 	else
-	{
-		SG_SWARNING("Trying to run GPU code without GPU backend registered.\n");
-		return vector;
-	}
+		SG_SERROR("No GPU backend registered.\n");
```

Comment:
@OXPHOS imagine that in some shogun algo we write some code like this
```c++
SGVector<T> a; //
auto b = linalg::to_gpu(a);
linalg::invoke_method_foobar(b);
auto c = linalg::from_gpu(b);
```
but say, some poor guy running shogun on his poor machine doesn't have a gpu backend.. for him, this above code shouldn't throw error..